window.py

```diff
@@ -81,3 +80,7 @@ def create_table_panel(self):
         self.table_panel = TablePanel(self.splitter, self.buttons)
         self.table_panel.Hide()
         self.panel_handler.table = self.table_panel
+
+    def to_grid(self, event):
+        if isinstance(self.FindFocus(), type(self.table_panel.grid)):
+            self.table_panel.grid.SetGridCursor(1, 0)
```

Comment:
este vyzera ze to sposobuje problemy ked sa pouzivatel prepne na tabulku pocas merania, asi by bolo dobre prepinanie na nu vtedy zakazat, tu by malo stacit spravit `if isinstance(self.FindFocus(), type(self.table_panel.grid)) and not self.cont_measurement` a [tu](https://github.com/TIS2019-FMFI/meranie-velicin/blob/d7b9c3b1f1d98125b17931360df9b5a33e1aed93/connection.py#L51) v ife pridat `self.handler.window.cont_measurement = False`